lms/djangoapps/discussion_api/api.py

```diff
@@ -49,6 +51,14 @@
 from openedx.core.lib.exceptions import CourseNotFoundError, PageNotFoundError
 
 
+class DISCUSSIONENTITY(Enum):
+    """
+    Enum for different types of discussion related entities
+    """
+    THREAD = 'thread'
+    COMMENT = 'comment'
```

Comment:
Use normal class/instance casing for `Enum` classes, as in the official examples here: https://docs.python.org/3/library/enum.html